test/integration/connect/envoy/case-centralconf/verify.bats

```diff
@@ -54,3 +54,23 @@ load helpers
     must_match_in_prometheus_response localhost:1234 \
     '[\{,]envoy_http_conn_manager_prefix="upstream_s2_http"[,}]'
 }
+
+function get_envoy_cluster_config {
+  local HOSTPORT=$1
+  local CLUSTER_NAME=$2
+  run retry_default curl -s -f $HOSTPORT/config_dump
+  [ "$status" -eq 0 ]
+  echo "$output" | jq --raw-output "
+    .configs[1].dynamic_active_clusters[]
+    | select(.cluster.name|startswith(\"${CLUSTER_NAME}\"))
+    | .cluster
+  "
+}
+
+@test "s1 proxy should have been configured with passive_health_check" {
```

Comment:
Another thought, I wonder whether this should go into a different integration tests, since these new config values are explicitly **NOT** centralized config, right?

If you wanted, you could probably rename `case-upstream-connection-limits` and put this check there.

Reply:
good idea! I didn't want to make another case for this. I think a renaming that case should work